### DISCUSSION TITLE
繩網能成為APP嗎?

### DISCUSSION THREAD

**Discussion by Lun603** [share121/inter-knot#5315] [其他]
繩網以後有機會成為app嗎?好希望能在apple store上下載!

**Comment by Lun603** [comment]
只是好奇，因為繩網在手機網頁上不太方便<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by stars52371** [comment]
这网站也就hexo前端+GitHub后端，不上苹果商店大概是因为成为Apple个人开发者得花钱，release里也发布了适用于iPhone的iPA安装包，教程在 #4893 上有
希望这能帮到你

**Reply by stars52371** [reply]
[在iOS设备上使用绳网app #4893](https://github.com/share121/inter-knot/discussions/4893)

**Reply by Lun603** [reply]
哇喔！謝謝！我今天回去試試！

**Comment by Yizakl** [comment]
这699没必要花
你可以转我2块试试pdd上的上架真不真（

**Reply by Yizakl** [reply]
还有，Apple Store是卖Apple产品的喵
App Store才是应用商店喵

**Reply by KELERSAMA** [reply]
大佬

**Comment by Psinary** [comment]
弄个网页快捷方式到桌面，浏览器隐藏UI，不就和APP类似了<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
